tensorboard/webapp/notification_center/_redux/BUILD

```diff
@@ -55,6 +55,7 @@ tf_ts_library(
     srcs = [
         "testing.ts",
     ],
+    visibility = ["//tensorboard:internal"],
```

Comment:
From which feature are you trying to use this target from?

Reply:
for testing in cl/373274039

Reply:
There is almost no point of providing the correct structure especially when the effects will be only using the public API (selector) and won't need to know much the store itself. Please omit this change and just use `store.overrideSelector` especially across the feature boundary.

Reply:
I see. Drop this pr.